app/services/rfd.local.server.ts

```diff
@@ -58,7 +65,17 @@ export function fetchLocalRfd(num: number): LocalRfd {
   }
 }
 
+function resolveInclude(num: number, path: string) {
+  checkForRelativePath(path)
+
+  const numStr = num.toString().padStart(4, '0')
```

Comment:
pass `numStr` so we don't have to do this, or even do this this entire function inline the loop